### PR TITLE
Added details on security steps as last step before sending payment

### PR DESCRIPTION
**Pull request by sbddesign**
I added some clarification around security steps to the Payments - Sending Bitcoin page. I reformatted the text and described how security confirmations should come as the last step in the "review and confirm" process. There is something that seems innately right to me that when the user goes through a security confirmation step as their final consent to pay.

For example:
- Don't ask the user to enter biometrics after scanning a lightning QR code but _before_ seeing the invoice details
- Don't ask the user to enter PIN and then ask them to select their transaction fees afterward

[💻 Deploy Preview](https://deploy-preview-672--sad-borg-390916.netlify.app/guide/payments/send/#review--approval)

### CODE REVIEW

**Comment by GBKS** [comment]
You think PIN entry is needed if the user has already authenticated via Face Id or Touch Id to access the phone in the first place?

**Comment by Bosch-0** [comment]
> You think PIN entry is needed if the user has already authenticated via Face Id or Touch Id to access the phone in the first place?

Yes, bitcoin payments should always require some kind of security at the moment of sending.

**Comment by moneyball** [comment]
I don't agree PIN entry is needed for small amounts if the user has already enabled Touch/Face ID for security. Only above a threshold amount T should a PIN be required IMO.

**Comment by sbddesign** [comment]
> You think PIN entry is needed if the user has already authenticated via Face Id or Touch Id to access the phone in the first place?

This discussion is shifting slightly to something like: _should PIN entry be required, and if so, what is its correct usage if combined with other security measures?_

The claim I am making with this PR is more narrow: if one chooses to put a security step in the payment flow, put it as the last step in the payment flow instead of jamming it awkwardly into the middle of the payment flow. The latter is confusing.

I've tried to update the text to more accurately reflect my intent, as well as other comments here.

To answer the question about if PIN is entry is needed, I think that depends. Many wallets (Muun, Breez, etc.) require PIN on opening the app but not before send. Some wallets (CashApp) do not require PIN on opening but do require before send. Casa requires PIN on opening and I _think_ requires PIN before send. These apps all target different types of users. Personally, I say pick a lane and do PIN on entry _or_ PIN on send but _not both_ (at least not for smaller amounts of money). 

I feel like deep-diving into the intricacies of this topic might be drifting outside the topic of this page and probably belongs on the [Protecting a Wallet](https://bitcoin.design/guide/onboarding/protecting-a-wallet/) page. That page outlines how choosing a security model is nuanced and depends on the audience. If that page is lacking crucial information, perhaps we should open an issue and discuss updates to that page.

**Comment by moneyball** [comment]
Re-reading this I don't think anyone was actually suggesting what I said "I don't agree with" :) So I retract that part but still think the latter portion of my statement should be the case, and I suspect others do too. The design guide should speak to this and provide best practices (how to think about when to require PIN vs. biometrics vs. nothing).

**Review by Bosch-0 — APPROVED** [review]
LGTM :)

File: guide/payments/send.md [hunk 1]
```diff
@@ -77,6 +77,14 @@ imagesErrors:
     - file: generic-error-details
       alt: A screen showing error details
       caption: Access to error details for problem-solving.
+imagesReview:
```
Comment (GBKS) [inline guide/payments/send.md:80]: Duplicate line here.

**Review by GBKS — CHANGES_REQUESTED** [review]
LGTM, except for that one duplicate line.

**Review by GBKS — APPROVED** [review]

**Pull request merged**